cmd/podspec-to-arm/main.go

```diff
@@ -99,46 +103,100 @@ func main() {
 			// read secrets from file
 			secretsMap := map[string]corev1.Secret{}
 			if k8secrets != "" {
-				secretsfile, err := ioutil.ReadFile(k8secrets)
+				secretsfile, err := os.ReadFile(k8secrets)
 				if err != nil {
-					fmt.Println(err)
+					fmt.Println("Error reading secrets file:", err)
 					return
 				}
 				err = yaml.Unmarshal([]byte(secretsfile), &secretsMap)
 				if err != nil {
-					fmt.Println("error unmarshalling secrets map")
-					fmt.Println(err)
+					fmt.Println("Error unmarshalling secrets:", err)
```

Comment:
Since this is an error, can we direct it to os.Stderr? https://stackoverflow.com/questions/29721449/how-can-i-print-to-stderr-in-go-without-using-log

Reply:
I think most print statements in this file are for errors. Do we want to change all of them? @ksayid

Reply:
Yeah, I think it makes sense to change them. Might as well exit 1 too?

Reply:
ok updated to print to stderr and have exit code 1 where necessary